config/config_builder/Cargo.toml

```diff
@@ -10,6 +10,7 @@ edition = "2018"
 bincode = "1.1.1"
 clap = "2.32"
 hex = "0.3.2"
+rand = "0.6.5"
```

Comment:
why not 0.7?

Reply:
Because it's better to upgrade in one fell swoop and I believe I wrote a script at some point (`scripts/cargo_update_outdated.sh`) and that thing hasn't told me it's time yet :)